site/pages/404.tsx

```diff
@@ -0,0 +1,15 @@
+import { GetStaticProps } from "next";
+import { Custom404 } from "components/pages/errors/Custom404";
+import { loadTranslation } from "lib/i18n";
+
+export const getStaticProps: GetStaticProps = async (ctx) => {
+  const translation = await loadTranslation(ctx.locale);
+  return {
+    props: {
+      translation,
+    },
+    revalidate: 60,
```

Comment:
we don't need to revalidate since this page will never change between builds

Reply:
done